src/index.html

```diff
@@ -5,8 +5,454 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header">
+      <a href="#home">
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="header__nav-logo"
+        >
+      </a>
+
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a
+              href="#apple"
+              class="header__nav-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="#samsung"
+              class="header__nav-link"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="#smartphones"
+              class="header__nav-link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="#laptops"
+              class="header__nav-link"
+              data-qa="nav-hover"
+            >
+              Laptops &amp Computers
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="#gadgets"
+              class="header__nav-link"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="#tablets"
+              class="header__nav-link"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="#photo"
+              class="header__nav-link"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="#video"
+              class="header__nav-link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="catalog main__catalog">
+        <div
+          class="card catalog__card"
+          data-qa="card"
+        >
+          <img
+            class="card__product-photo"
+            src="images/imac.jpeg"
+            alt="APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)"
+          >
+
+          <h2 class="card__product-name">
+            APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__product-code">
+            Product code: 195434
+          </p>
+
+          <div class="card__stars">
+            <div class="card__block stars stars__star--4">
```

Comment:
Remove unused  classes
```suggestion
            <div class="stars stars__star--4">
```